dashboard/cron.py

```diff
@@ -676,8 +680,14 @@ def do(self) -> str:
         else:
             logger.warn("data_last_updated not updated because of an Exception during this run")
 
-        status += "End cron: " + str(datetime.now()) + "\n"
 
-        logger.info("************ total status=" + status + "\n")
+        if settings.LRS_IS_BIGQUERY:
+            total_tbytes_billed = self.total_bytes_billed / 1024 / 1024 / 1024 / 1024
+            # $6.25 per TB as of Feb 2024 https://cloud.google.com/bigquery/pricing
```

Comment:
We need to watch for the cost. I have 20 MyLA courses in localhost, and the initial run with this PR cost $18, and subsequent run cost $0.1

Reply:
I only has 4 course and costed $24 next run cost $0.1. I would expect to cost less
```
// This is the total row count
resources_access:	2115
resources:	248
submissions:	1759
assignments:	95
user:	100
course:	5
```

Reply:
The cost is high likely have courses without a start date, without a term or with a really old term in your database. It might also be a bug with retrieving the dates for terms which wouldn't be good. The event store is getting huge now so doing a full query is multiple TB of data. 

I think we should have a safety case in here so it doesn't retrieve more than X months of data from the `event_store` unless explicitly directed to. Generally there won't ever be a case to populate old course data.